### ISSUE TITLE
Menu in settings.js with uid in path

### ISSUE THREAD

**Issue by chrisschaub** [signalpoint/DrupalGap#344]
What's the best way to get a menu path like
user/%/edit
to work in settings.js. I'm creating a custom menu for viewing and editing user profiles and need the links to work. Can't seem to get the % to be replaced with the user's uid. 

Thanks!

**Comment by signalpoint** [comment]
@chrisschaub When dynamic arguments are needed in menus, they will not live in the settings.js file. Instead, this is what "local tasks" are for.

I happened to be implementing mass fixes for local tasks today. Grab the latest bin/drupalgap.js for the fixes.

I also created documentation outlining what local tasks are, and how to use them:

http://drupalgap.org/node/294

Here's a related issue which also contains some example code of some one else's local tasks and how they implemented them for the user id in the path: #301 .
